### PR TITLE
feat: add neutral_token drops data

### PR DESCRIPTION
**Pull request by Antosik**
Adds the ability to track the neutral tokens drops
Related odota/parser PR - https://github.com/odota/parser/pull/72

### CODE REVIEW

File: sql/create_tables.sql [hunk 1]
```diff
@@ -137,7 +137,8 @@ CREATE TABLE IF NOT EXISTS player_matches (
   roshans_killed int,
   observers_placed int,
   party_size int,
-  net_worth int
+  net_worth int,
+  neutral_tokens_log json[]
```
Comment (howardchung) [inline sql/create_tables.sql:141]: cql also needs to be updated

File: sql/create_tables.cql [hunk 2]
```diff
@@ -66,6 +66,7 @@ CREATE TABLE IF NOT EXISTS player_caches (
   item_neutral text,
   average_rank int,
   hero_variant text,
+  neutral_tokens_log text,
```
Comment (howardchung) [inline sql/create_tables.cql:69]: player_matches not player_caches

Comment (howardchung) [reply sql/create_tables.cql:69]: Oh wait we recently switched to the blob model. So this change is actually not needed in the cql

**Comment by Antosik** [comment]
@howardchung 🙏

**Review by howardchung — APPROVED** [review]

**Pull request merged**